app/models/concerns/users/info.rb

```diff
@@ -18,11 +18,21 @@ def member_at?(organization)
       organization_user_at(organization).present?
     end
 
-    def organizations_with_admin_access
+    def organizations_with_access
       if super_admin?
-        @organizations_with_admin_access ||= Organization.all
+        @organizations_with_access ||= Organization.all
       else
-        organizations.select { |org| admin_at?(org) }
+        organizations
+      end
+    end
+
+    def organizations_with_permission(permission)
```

Comment:
@smellsblue What is the difference between organizations_with_access and organizations_with_permission?